fe/fe-core/src/main/java/org/apache/doris/load/BrokerFileGroup.java

```diff
@@ -237,7 +238,8 @@ public void parse(Database db, DataDescription dataDescription) throws DdlExcept
             jsonPaths = dataDescription.getJsonPaths();
             jsonRoot = dataDescription.getJsonRoot();
             fuzzyParse = dataDescription.isFuzzyParse();
-            readJsonByLine = dataDescription.isReadJsonByLine();
+            readJsonByLine = true;
```

Comment:
Add comment:
"For broker load, we only support reading json format data line by line, so we set `readJsonByLine` to true here."

Reply:
done